hw1/02-palindrome.html

```diff
@@ -1,27 +1,40 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<head>
+    <meta charset="UTF-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <title>02 - Palindrome</title>
     <link
-      rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+            rel="stylesheet"
+            href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"/>
     <script defer src="02-palindrome.js"></script>
-  </head>
-  <body>
-    <main>
-      <section>
-        <h1>Exercise 02 - Palindrome</h1>
-        <div>
-          <label>Enter a positive number:</label>
-          <input type="number" />
+</head>
+<body class="vh-100"> <!-- Set body height to 100% of the viewport -->
+<div class="h-100 d-flex flex-column bg-primary m-0">
+    <main class="container-fluid m-0 p-0">
+        <div class="row justify-content-center">
+            <div class="col-auto">
+                <div class="card mt-4">
```

Comment:
I believe you can combine the class elements into a single `<div>` instead of using separate`<div> `for each.
for example `div class ="col-auto card mt-4`
This would help reduce the amount of code and make maintenance easier.

Reply:
Thank you! You're right. I'll update it in my next commit.